script/Arcueid/Day 8/08_00_ARC08_2.txt

```diff
@@ -151,7 +151,7 @@ Exactly who is that woman to you, Nii-san?"
 [sha:ca568c71b16e56f2aac207a232cd23fe4fbfda96]{
 -- Page 65, Offset 19839.
 -- ええ、私だって兄さんが将来どなたと関係を持とうと意見する気はありませんけど、兄さんはまだ学生でしょう？」
-Yes, I have no opinion on who you choose to have a future relationship with, but isn't it true that you're still a student?"
+Yes, I have no say on who you choose to have a future relationship with, but isn't it true that you're still a student?"
```

Comment:
As I understand it, ikensuru is more like give opinions/views, which would mean the idea here is more like "I'll spare you my thoughts on who I think you should be getting in relationships with" / "Far be it from me to tell you who to...", whereas 'say' in this context reads more about power, and whether she could _control_ who he gets in a relationship with.